extensions/amp-analytics/0.1/iframe-transport.js

```diff
@@ -273,23 +272,23 @@ export class IframeTransport {
   /**
    * Removes all knowledge of cross-domain iframes.
    * Does not actually remove them from the DOM.
-   * @VisibleForTesting
+   * @visibleForTesting
    */
   static resetCrossDomainIframes() {
     IframeTransport.crossDomainIframes_ = {};
   }
 
   /**
    * @return {string} Unique ID of this instance of IframeTransport
-   * @VisibleForTesting
+   * @visibleForTesting
    */
   getCreativeId() {
     return this.creativeId_;
   }
 
   /**
    * @return {string} Type attribute of parent amp-analytics instance
-   * @VisibleForTesting
+   * @visibleForTesting
```

Comment:
I'm curious if the old (mis-spelt) annotations prevented any tests from working. I'm also curious if this makes a difference in the resulting minified bundle size, now that the correct annotations are in place.

Reply:
I don't think the annotation will affect test behavior. Don't know if it affect bundle size though.

Reply:
With this change: 77.19KB (https://travis-ci.org/ampproject/amphtml/jobs/385432514#L648)
Before this change: 77.2KB (https://travis-ci.org/ampproject/amphtml/jobs/385432417#L648)

Not sure if that's a real decrease or just noise.